### DISCUSSION TITLE
Iceberg parallel partitioned inserts?

### DISCUSSION THREAD

**Discussion by nfoerster** [trinodb/trino#13588] [Q&A]
I have a question regarding the possibilities of parallel iceberg inserts when the data is partitioned. I tried it and had a lot of concurrency errors, I think because of problems during metadata write operation in the Iceberg itself. I'm just thinking if there is a setting or parameter to enable this feature. 

```
insert into iceberg.test.table_test
     select * from iceberg.test.view_test where msgid={msgid}
```

**Comment by raunaqmorarka** [comment]
Does using `set session preferred_write_partitioning_min_number_of_partitions=1;` help here ?
